discovery/gce/gce.go

```diff
@@ -52,15 +52,13 @@ const (
 var (
 	gceSDRefreshFailuresCount = prometheus.NewCounter(
 		prometheus.CounterOpts{
-			Namespace: namespace,
```

Comment:
👍 for the change. I think you removed all usages of this in this PR so we can remove the [var](https://github.com/prometheus/prometheus/blob/d67dd561af2cd01d08724c9007d86603891d1261/discovery/gce/gce.go#L49) all together.